src/languages/php.md

```diff
@@ -62,73 +66,107 @@ You can disable those by adding them to the `disabled_extensions` list.
+| igbinary     |     |     |     | *   | *   |
+| imagick      | *   | *   | *   | *   | *   |
+| imap         | *   | *   | *   | *   | *   |
+| interbase    | *   | *   | *   | *   | *   |
+| intl         | *   | *   | *   | *   | *   |
+| json         |     |     | *   | *   | *   |
+| ldap         | *   | *   | *   | *   | *   |
+| mcrypt       | *   | *   | *   | *   | *   |
+| memcache     | *   | *   | *   |     |     |
+| memcached    | *   | *   | *   | *   | *   |
+| mongo        | *   | *   | *   |     |     |
+| mongodb      |     |     |     | *   | *   |
+| msgpack      |     |     | *   | *   | *   |
+| mssql        | *   | *   | *   |     |     |
+| mysql        | *   | *   | *   |     |     |
+| mysqli       | *   | *   | *   | *   | *   |
+| mysqlnd      | *   | *   | *   |     |     |
+| odbc         | *   | *   | *   | *   | *   |
+| opcache      |     | *   | *   | *   | *   |
+| pdo          | *   | *   | *   | *   | *   |
+| pdo_dblib    | *   | *   | *   | *   | *   |
+| pdo_firebird | *   | *   | *   | *   | *   |
+| pdo_mysql    | *   | *   | *   | *   | *   |
+| pdo_odbc     | *   | *   | *   | *   | *   |
+| pdo_pgsql    | *   | *   | *   | *   | *   |
+| pdo_sqlite   | *   | *   | *   | *   | *   |
+| pecl-http    |     |     | *   |     |     |
+| pgsql        | *   | *   | *   | *   | *   |
+| pinba        | *   | *   | *   |     |     |
+| propro       |     |     | *   |     |     |
+| pspell       | *   | *   | *   | *   | *   |
+| pthreads     |     |     |     |     | *   |
+| raphf        |     |     | *   |     |     |
+| readline     | *   | *   | *   | *   | *   |
+| recode       | *   | *   | *   | *   | *   |
+| redis        | *   | *   | *   | *   | *   |
+| snmp         | *   | *   | *   | *   | *   |
+| sockets      |     |     | *   | *   | *   |
+| spplus       | *   | *   |     |     |     |
+| sqlite3      | *   | *   | *   | *   | *   |
+| ssh2         | *   | *   | *   |     |     |
+| tidy         | *   | *   | *   | *   | *   |
+| xcache       | *   | *   |     |     |     |
+| xdebug       | *   | *   | *   | *   | *   |
+| xhprof       | *   | *   | *   |     |     |
+| xmlrpc       | *   | *   | *   | *   | *   |
+| xsl          | *   | *   | *   | *   | *   |
+| zendopcache  | *   |     |     |     |     |
 
 
 > **note**
 > You can check out the output of `ls /etc/php5/mods-available` to
 > see the up-to-date complete list of extensions after you SSH into
-> your environment. For PHP 7, use `ls /etc/php/mods-available`.
+> your environment. For PHP 7, use `ls /etc/php/*/mods-available`.
+
+## Alternate start commands
+
+PHP is most commonly run in a CGI mode, using PHP-FPM.  That is the default on Platform.sh.  However, you can also start alternative processes if desired, such as if you're running an Async PHP daemon, a thread-based worker process, etc.  To do so, simply specify an alternative start command in `platform.app.yaml`, similar to the following:
```

Comment:
I'm being generic here, to refer to CGI in the abstract (aka, the web as if it were 1991).  PHP userspace can't really tell the difference between mod_php and FastCGI.  It definitely knows the difference between that and CLI, however.